### PR TITLE
[Test] Changed the linear solver type in Parabolic/TwoPhaseFlowPrho/MoMaS/Twophase_MoMaS_quad.prj

### PR DESCRIPTION
**Pull request by wenqing**
To fix the test of LARGE_2D_TwoPhase_Prho_MoMa (see PR #2056).

### CODE REVIEW

File: Tests/Data/Parabolic/TwoPhaseFlowPrho/MoMaS/Twophase_MoMaS_quad.prj [hunk 1]
```diff
@@ -259,7 +259,7 @@
             <name>general_linear_solver</name>
             <lis>-i bicgstab -p ilu -tol 1e-18 -maxiter 10000</lis>
             <eigen>
-                <solver_type>BiCGSTAB</solver_type>
+                <solver_type>SparseLU</solver_type>
                 <precon_type>ILUT</precon_type>
                 <max_iteration_step>10000</max_iteration_step>
                 <error_tolerance>1e-18</error_tolerance>
```
Comment (endJunction) [inline Tests/Data/Parabolic/TwoPhaseFlowPrho/MoMaS/Twophase_MoMaS_quad.prj]: Add <scaling>, remove preconditioner, iterations, and tolerance.

Comment (endJunction) [reply Tests/Data/Parabolic/TwoPhaseFlowPrho/MoMaS/Twophase_MoMaS_quad.prj]: https://github.com/ufz/ogs/pull/2056 see the commit before last.

Comment (wenqing) [reply Tests/Data/Parabolic/TwoPhaseFlowPrho/MoMaS/Twophase_MoMaS_quad.prj]: I see.

**Comment by endJunction** [comment]
@wenqing Added the change to the #2149 Cleanup. The other PR will take a while...

**Pull request closed without merging**